tests/test_default_qubit.py

```diff
@@ -596,11 +596,11 @@ def test_unsupported_observables(self):
             def circuit(*x):
                 """Test quantum function"""
                 x = prep_par(x, op)
-                return op(*x, wires=wires)
```

Comment:
How come this worked before with no `expval`?

Reply:
`op` _was_ the `expval` (see line 588 `op = getattr(qml.expval, g)`) 🙂 

After this change, `op` is a bare observable, so we need to explicitly wrap it in an `expval` function.